### PR TITLE
Change "hostname" in API to "URL" 

### PR DESCRIPTION
**Pull request by sjanota**
**Description**

Changes proposed in this pull request:

- Create url-link component to display links with copy and open
- Add getURL function to build url from parts
- Use those in APIs list, API details and service APIs list 

**Related issue(s)**

### CODE REVIEW

File: core/src/app/shared/services/generic-helpers.service.ts [hunk 1]
```diff
@@ -1,7 +1,26 @@
...
 @Injectable()
 export class GenericHelpersService {
-  public getHostnameURL = (hostname: string, domain = ''): string =>
-    `https://${hostname}${domain}`;
+  public getURL = ({
+    schema = 'https',
+    host,
+    domain = AppConfig.domain,
+    port = '',
+    path = '/'
```
Comment (dariadomagala) [inline core/src/app/shared/services/generic-helpers.service.ts]: What is it for?

File: core/src/app/shared/services/generic-helpers.service.ts [hunk 2]
```diff
@@ -1,7 +1,26 @@
 import { Injectable } from '@angular/core';
+import { AppConfig } from '../../app.config';
 
 @Injectable()
 export class GenericHelpersService {
-  public getHostnameURL = (hostname: string, domain = ''): string =>
-    `https://${hostname}${domain}`;
+  public getURL = ({
+    schema = 'https',
```
Comment (dariadomagala) [inline core/src/app/shared/services/generic-helpers.service.ts]: Maybe `protocol`?

Comment (sjanota) [reply core/src/app/shared/services/generic-helpers.service.ts]: URL it's called "schema".

File: core/src/app/shared/services/generic-helpers.service.ts (hunk 1 above)
Comment (sjanota) [reply core/src/app/shared/services/generic-helpers.service.ts]: When calling this method you may provide a path. I'm not using it now, but it was just so easy to add :D

File: core/src/app/shared/services/generic-helpers.service.ts (hunk 2 above)
Comment (dariadomagala) [reply core/src/app/shared/services/generic-helpers.service.ts]: 👍 
It's `scheme` but nvm 😄

**Pull request merged**